pages/en/lb3/Extend-your-API.md

```diff
@@ -144,7 +147,34 @@ You'll get the response:
 {
   "name": "Name of coffee shop is Bel Cafe"
 }
+
+```
+## accessing the remote method through path as source parameter 
+```
+
+ CoffeeShop.remoteMethod (
+        'getName',
+        {
+          http: {path: '/:id/getname', verb: 'get'},
+          accepts: {arg: 'id', type: 'number', http: { source: 'path' } },
```

Comment:
I find this rather unusual. If you want to write a method that's exposed at `/api/CoffeeShops/1/getname`, then the usual solution is to put that method on the model prototype.

```js
CoffeeShop.prototype.getName = function(cb) {
  // (you can access the id via "this.id")
  const response = "Name of coffee shop is " + instance.name;
  cb(null, response);
  console.log(response);
});

CofeeShop.remoteMethod('prototype.getName', {
  http: {path: '/getname', verb: 'get'},
  returns: {arg: 'name', type: 'string'},
});
```

This way LoopBack calls `findById` for you and takes care of edge cases like when there is no model instance with the given `id`.